tests/test_tempered_smc.py

```diff
@@ -130,15 +130,15 @@ def body_fn(carry, lmbda):
 ######################################
 
 
-def normal_potential_fn(x, chol_cov):
+def normal_logprob_fn(x, chol_cov):
     """multivariate normal without the normalizing constant"""
```

Comment:
Is it really no normalizing constant here?

Reply:
@AdrienCorenflos ?